src/sections/section4/Section4.css

```diff
@@ -0,0 +1,33 @@
+.codeutsava__section4 {
+    padding-top: 2rem;
+    padding-bottom: 2rem;   
+    font-family: 'Poppins'; 
```

Comment:
Different fonts are being used in different sections of the card, so please refer to figma design as they are using "Roboto" for paragraphs.